test/controllers/RegistrationControllerSpec.scala

```diff
@@ -238,7 +240,26 @@ class RegistrationControllerSpec extends VoaPropertyLinkingSpec with MockitoSuga
     val res =
       testRegistrationController(userDetails(confidenceLevel = ConfidenceLevel.L200)).submitIndividual()(fakeRequest)
     status(res) shouldBe SEE_OTHER
-    redirectLocation(res) shouldBe Some(s"/business-rates-property-linking/create-success?personId=$personId")
+    redirectLocation(res) shouldBe Some(s"/business-rates-property-linking/create-confirmation?personId=$personId")
+  }
+
+  "Call confirmation" should "return an valid page" in new SubmitOrganisation {
+    when(mockRegistrationService.continue(any(), any())(any(), any()))
+      .thenReturn(Future.successful(Some(RegistrationSuccess(personId))))
+
+    val res =
+      testRegistrationController(userDetails(confidenceLevel = ConfidenceLevel.L200)).confirmation(100)(fakeRequest)
+    status(res) shouldBe OK
+    val html = HtmlPage(res)
+    //Page title
+    html.shouldContainText("Registration successful")
+    html.shouldContainText("Your VOA personal ID number")
+    //Page should contains VOA Person ID value 100
+    html.shouldContainText("100")
```

Comment:
I think this test is not quite good enough. There could be a variety of other things on the page that will include the number 100. e.g. "You have 100 properties", or check reference "CHK1002345" or just some sort of css class could be "padding-top-100" or something like that... just checking if the HTML contains 100 (or 300) is not good enough test. Use a bit more text maybe to check it's in the page?

Reply:
I have updated the test